src/commands/new/selector.ts

```diff
@@ -16,6 +15,7 @@ module.exports = {
       print,
       parameters,
       template: { generate },
+      utils,
       args
     } = toolbox;
 
```

Comment:
Will need to change below as well:
```suggestion
      const filters = JSON.parse(parameters.options.filters || '[]');
```

Reply:
Done